damus/Views/ConfigView.swift

```diff
@@ -113,6 +113,11 @@ struct ConfigView: View {
                         }
                     }
                 }
+
+                Section("Left Handed") {
```

Comment:
I'm in the process of getting all bundled strings localized in a different PR. Would you mind wrapping the string here and below with an `NSLocalizedString`, and also add a comment explaining what the string is?

e.g.
```swift
Section(NSLocalizedString("Left Handed", comment: "<some comment explaining what this is>") {
    Toggle(NSLocalizedString("Left Handed", comment: "<some comment explaining what this is>"), isOn: $user_settings.left_handed)
        .toggleStyle(.switch)
}
```

Reply:
Absolutely! I'll get this done tonight when I'm back at my desktop.